updateclang/extract.sh

```diff
@@ -8,7 +10,11 @@ jextract --output ../src/main/java \
   @clang.symbols \
   ${LIBCLANG_HOME}/include/clang-c/Index.h
 
+echo "Adding copyrights..."
+
 for x in ../src/main/java/org/openjdk/jextract/clang/libclang/*.java; do
-head -$COPYRIGHTLEN $x | diff cp_header.txt - || ( ( cat cp_header.txt; echo; cat $x) > /tmp/file;
-mv /tmp/file $x )
+(cat cp_header.txt; echo; cat $x) > /tmp/file;
```

Comment:
on my machine, the script hang here. In reality we don't need this complex logic, so I've tweaked the instructions so that the libclang folder is always deleted first (this would also ensure that there's no files leftover from previous runs).